src/style.css

```diff
@@ -2,0 +1,81 @@
+@font-face {
+  font-family: 'Avenir Book';
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy';
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar {
+  font-family: Avenir, sans-serif;
+  width: 100%;
```

Comment:
Avoid fixing container size if there is no such requirement. Let the content size dictate it. To avoid overflow or accidental scroll bar.